.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 jobs:
     terraform:
         docker:
-            - image: patrickcpierson/terraform-aws:latest
+            - image: zboldyga/terraform-aws:latest
```

Comment:
what is the change here?

Reply:
updated to latest terraform, a subversion update

Reply:
needed to do it due to a bug in terraform